rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -252,6 +252,11 @@
       "min_compatible_shard_node":{
         "type":"string",
         "description":"The minimum compatible version that all shards involved in search should have for this request to be successful"
+      },
+      "enable_fields_emulation":{
```

Comment:
Is it really needed to publish this option here ? I thought it was used to validate yml tests so it could be removed ?

Reply:
I'll check if tests pass if I remove it again. I think I added it thinking about clients, not sure which one Discovery uses and if they are able to use the option if its not in the specs.

Reply:
Test passed without this, I'll leave it at this for the time being but we might need it if we introduce yaml tests using this flag or if the clients need it for some reason.